generator/src/main/java/org/javabubble/generator/model/ModelLoader.java

```diff
@@ -13,23 +13,23 @@
 
 public class ModelLoader {
 
-	private final Path base;
+	private final Path path;
 	private final ObjectMapper mapper;
 
-	public ModelLoader(Path base) {
-		this.base = base;
+	public ModelLoader(Path path) {
+		this.path = path;
```

Comment:
The idea here was that the model may be loaded from multiple yaml files in the future.

Reply:
I see, but this allows better testability with small test sets for the moment. 
Should we refactor to a base directory with separate files if this idea becomes more clear?